client.go

```diff
@@ -255,11 +255,7 @@ func connectViaUnix(c *Client, remote *RemoteConfig) error {
 		//   unix:///path/to/socket
 		//   unix:/path/to/socket
 		//   unix:path/to/socket
-		path := strings.TrimPrefix(remote.Addr, "unix:")
-		if strings.HasPrefix(path, "///") {
-			// translate unix:///path/to, to just "/path/to"
-			path = path[2:]
-		}
+		path := strings.TrimPrefix(strings.TrimPrefix(remote.Addr, "unix:"), "//")
```

Comment:
This is necessary to allow relative paths for the server, which appear to be allowed given the 'unix:path/to/socket' example above.  If relative paths are not allowed then explicit checks should be added here and in remote.go.  Stripping "unix:" without stripping the "//" prepended by addServer results in relative paths being inadvertently converted to absolute paths.